build.gradle

```
@@ -81,15 +84,76 @@ if (ciBuild) {
     version = "$major.$minor-SNAPSHOT"
 }
 
+group = 'org.mpsqa'
+
 configurations {
     mps
-    languageLibs
     // includes also junit tasks support
     antLib
+    
+    plantuml
```

You can configure each configuration as non-transitive to avoid disabling transitivity for each dependency:
```suggestion
    plantuml { transitive = false }
```